packages/@lwc/perf-benchmarks/README.md

```diff
@@ -61,3 +61,14 @@ BENCHMARK_TIMEOUT=5
 See the [Tachometer documentation](https://github.com/Polymer/tachometer) for details on what these mean.
 
 If anything gets messed up when comparing to the other branch, add the `--force-clean-npm-install` flag when running `tach`.
+
+Also note that, due to how NX does caching, any environment variables you pass to `build:performance` should also
+be passed to `test:performance`. Otherwise, `test:performance` will use whatever environment variables you give it
+(or don't give it) to re-run `build:performance`.
+
+## Best benchmarks
+
+This package also supports [Best](https://bestjs.dev) as a benchmark runner. To run the Best tests, you must run
+`yarn build:performance` as usual. To run the tests locally, run:
+
+    yarn test:performance:best
```

Comment:
Same observation as above.

Reply:
Because we're not using NX for this, it doesn't depend on `build:performance`. But it should.

Maybe I should sit down and actually figure out how to configure NX for this.

Reply:
OK I figured it out. The Best `--runner` CLI arg was conflicting with NX's own `--runner` CLI arg. Adding an explicit `yarn test:performance:best:ci` fixed it.